charts/cluster-api-runtime-extensions-nutanix/templates/cluster-autoscaler/manifests/helm-addon-installation.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
 data:
   values.yaml: |-
     ---
-    fullnameOverride: "cluster-autoscaler-{{ `{{ .Cluster.Name }}` }}"
+    fullnameOverride: "cluster-autoscaler-{{ `{{ md5sum (printf "%s/%s" .Cluster.Namespace .Cluster.Name) }}` }}"
```

Comment:
~Good to know we have access to some of these funcs~, ahh I see where its defined. Great use of template funcs!